src/main/java/uk/gov/hmcts/reform/sscs/config/ObjectMapperConfig.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.sscs.config;
+
+import static com.fasterxml.jackson.databind.DeserializationFeature.READ_ENUMS_USING_TO_STRING;
+import static com.fasterxml.jackson.databind.DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE;
+import static com.fasterxml.jackson.databind.SerializationFeature.WRITE_ENUMS_USING_TO_STRING;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+
+public class ObjectMapperConfig {
+
+    private ObjectMapperConfig() {
+        //
+    }
+
+    public static ObjectMapper mapper() {
```

Comment:
This exact same helper seems to be copy pasted into every project - can just make the identical uk.gov.hmcts.reform.sscs.config.SpringConfig.mapper() public and use that